types/meteor/index.d.ts

```diff
@@ -2145,3 +2145,67 @@ declare module "meteor/accounts-base" {
         function onLogout(func: (user: Meteor.User, connection: Meteor.Connection) => void): void;
     }
 }
+
+declare module "meteor/webapp" {
+
+    import * as http from "http";
+    import * as connect from "connect";
+    interface StaticFiles {
+        [key: string]: {
+            content?: string,
+            absolutePath: string,
+            cacheable: boolean,
+            hash: string,
+            sourceMapUrl?: string,
+            type: string
+        }
+    }
+    module WebApp {
```

Comment:
Would be awesome though if you added a couple of basic tests just to check that main definitions exist etc